fail2ban/protocol.py

```diff
@@ -55,6 +55,7 @@
 ["start <JAIL>", "starts the jail <JAIL>"], 
 ["stop <JAIL>", "stops the jail <JAIL>. The jail is removed"], 
 ["status <JAIL>", "gets the current status of <JAIL>"],
+["status <JAIL> extended", "gets the current status of <JAIL> with extended info"],
```

Comment:
That is fair, I was thinking extended would be any additional info, and someone could add additional output to the same 'extended' status from any source (all calls to Cymru are made once, then calls to parse the data). Therefore someone could add GeoIP sourced data with a method `geBanListExtendedCountryGeoIP` and add that to the existing `statusExtended` method's return.